controller/source/controller.cpp

```diff
@@ -25,10 +30,12 @@ auto handle_get(const std::unique_ptr<kv_client> &client,
   // if the key exists and complies with the gdpr rules
   // then return the value of the get operation
   if (filter.validate()) {
+    logger->log(query_args, res.has_value());
     // TODO: write the value to the client socket
     assert(res);
   } 
   else {
+    logger->log(query_args, false, "Filter validation failed.");
```

Comment:
We definitely need shorter message.
I understand, it's easier to understand, but would be better to map these messages to a combination of bits (or an enum) that we can use to convert to meaningful messages when reading the log.

Reply:
I removed logs on policy compliance failures - logging only on valid queries. Also, made the query operation an enum. Please see the PR description for sample logs. 

Please let me know if we should/can further reduce the length.